csrc/pf_guts.h

```diff
@@ -26,8 +26,18 @@
 ** PFORTH_VERSION changes when PForth is modified.
 ** See README file for version info.
 */
-#define PFORTH_VERSION_CODE 32
-#define PFORTH_VERSION_NAME "2.1.0"
+#define PFORTH_VERSION_CODE 33
+#define PFORTH_VERSION_NAME "2.1.1"
+
+/* Most Forth word names are short - so 31 characters should be enough.
+ * However if you are integrating with other systems - for example libSDL
+ * some names are longer than 31 characters. This provides up to 63 characters 
+ * for a word names.
+ *
+ * Long term this can become the default since there is no storage penalty
+ * when not using them.
+ */
+#define LONG_NAME_SUPPORT
```

Comment:
Let's make this like the other compile time switches defined at:

https://www.softsynth.com/pforth/pf_ref.php#Compiler-Options

Let's rename it to PF_SUPPORT_LONG_NAMES
and leave it undefined for now.
Maybe we can test it on/off in GitHub Actions.